sklearn/model_selection/tests/test_validation.py

```diff
@@ -857,13 +855,12 @@ def split(self, X, y=None, groups=None):
 
     X, y = load_iris(return_X_y=True)
 
-    warning_message = ('Number of classes in training fold (2) does '
-                       'not match total number of classes (3). '
+    warning_message = ('Number of classes in training fold \\(2\\) does '
+                       'not match total number of classes \\(3\\). '
```

Comment:
I think  you can use `r'Number...` and then single `\` escaping of the parenthesis.

Reply:
Thanks for the suggestion. I fixed the code.

Reply:
@ogrisel 

I found that the suggested code could not pass the flake8.
https://app.circleci.com/pipelines/github/scikit-learn/scikit-learn/11838/workflows/0a4f68fc-8b96-42e3-893b-2248fbe16096/jobs/132836

I'm not sure why flake8 could not pass only this line.
Do we need to disable flake8 check for this line? Or any other suggestions?